README.md

```diff
@@ -18,7 +18,11 @@ import NukeWebPPlugin
 
 let imageView = UIImageView()
 let webpimageURL = URL(string: "https://example.com/sample.webp")!
-WebPImage.manager.loadImage(with: webpimageURL, into: imageView)
+Nuke.ImageDecoderRegistry.shared.register { (context) -> ImageDecoding? in
```

Comment:
It would probably make sense to add a convenience global function like `WebPImageDecoder.enable()` to simplify installation steps.

Reply:
@kean It was necessary to think and implement things on the side you used ... :(

Reply:
I mean something like this:

```swift
public func enable() { // Global function
    Nuke.ImageDecoderRegistry.shared.register { (context) -> ImageDecoding? in
        return WebPImageDecoder.isWebPFormat(data: context.data) ? WebPImageDecoder() : nil
    }
}
```

in README:

```swift
NukeWebPPlugin.enable()
Nuke.loadImage(with: url, into: imageView)
```

I think it looks more user-friendly then asking users to copy-paste this code.